addons/expanding-search-bar/expanding-search-bar.js

```diff
@@ -0,0 +1,69 @@
+export default async function ({ addon, global, console }) {
+
+	///scratch-www checking
+	var exsearch_isWWW; //True if scratch-www, false if not
+	if (document.getElementsByClassName("site-nav")[0]) {
+		exsearch_isWWW = false;
+	} else {
+		exsearch_isWWW = true;
+	}
```

Comment:
You probably don't want to check once, since scratchr2 is an addon. I would use `waitForElement` and a while loop

Reply:
Assuming you're talking about the scratch-www detection, changed the script to use the client version API.

Reply:
So should I wait for the search bar to load before doing the stuff after?

Reply:
Anyways, waited to load the search bar.